src/explore-education-statistics-admin/src/pages/users/ManageUserPage.tsx

```diff
@@ -34,20 +33,25 @@ const updateRoleFormErrorMappings = [
   }),
 ];
 
-interface AddReleaseRoleFormValues {
-  selectedReleaseId: string;
-  selectedReleaseRoleId: string;
-}
-
 const addReleaseFormErrorMappings = [
   mapFieldErrors<AddReleaseRoleFormValues>({
-    target: 'selectedReleaseRoleId',
+    target: 'selectedReleaseRole',
     messages: {
       USER_ALREADY_HAS_RELEASE_ROLE: 'The user already has this release role',
     },
   }),
 ];
 
+const addPublicationFormErrorMappings = [
+  mapFieldErrors<AddPublicationRoleFormValues>({
+    target: 'selectedPublicationRole',
+    messages: {
+      USER_ALREADY_HAS_PUBLICATION_ROLE:
```

Comment:
On the back end I've renamed this to `USER_ALREADY_HAS_RESOURCE_ROLE`  to be used for both Releases and Publications. I didn't realise it was used on the front end.

Reply:
This is changed now.